apollo-router/src/plugins/coprocessor.rs

```diff
@@ -161,12 +161,31 @@ pub(super) struct RouterConf {
     pub(super) body: bool,
     /// Send the SDL
     pub(super) sdl: bool,
+    /// Send the uri
+    pub(super) uri: bool,
+    /// Send the method
+    pub(super) method: bool,
 }
 
+/// What information is passed to a router request/response stage
+#[derive(Clone, Debug, Default, Deserialize, PartialEq, Serialize, JsonSchema)]
+#[serde(default, deny_unknown_fields)]
+pub(super) struct RouterResponseConf {
+    /// Send the headers
+    pub(super) headers: bool,
+    /// Send the context
+    pub(super) context: bool,
+    /// Send the body
+    pub(super) body: bool,
+    /// Send the SDL
+    pub(super) sdl: bool,
+    /// Send the http status
```

Comment:
nit/not-for-now: These are just comments, so we can follow-up on this later, but going forward, I would suggest we **consistently** capitalize our abbreviations (e.g., above you've introduced `SDL`, below we had an existing `URI`, you've introduced `http` here, but we also have `uri` introduced above).  I think this will probably stand best as a documentation pass at this point, but we might want to stop the bleeding by being more prudent in our PR reviews about these important aesthetic things?

```suggestion
    /// Send the HTTP status
```